index.html

```diff
@@ -3,69 +3,103 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>OpenSeaDragon Parallel Images Example</title>
+    <title>OpenSeaDragon Split View Example</title>
     <style>
-        #openseadragon {
+        body, html {
+            margin: 0;
+            height: 100%;
+            overflow: hidden;
+        }
+        #viewer-container {
+            position: relative;
             width: 100%;
-            height: 600px;
+            height: 100%;
         }
-        .image-buttons {
-            margin: 10px 0;
+        #viewer {
+            position: absolute;
+            height: 100%;
+            width: 100%;
         }
-        .image-buttons button {
-            margin-right: 10px;
+        #divider {
+            position: absolute;
+            top: 0;
+            bottom: 0;
+            width: 2px;
+            background: black;
+            pointer-events: none;
```

Comment:
The `overlay_view` branch sets `cursor: ew-resize` here, using the "east-west resize" cursor style when the mouse is over the divider. I removed that style line since this version of the divider just always follows the mouse, not requiring a click and drag (as you'd usually need for a resizing element).